http-auth/routes/routes.go

```diff
@@ -0,0 +1,109 @@
+package routes
+
+import (
+	"fmt"
+	"html"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/joho/godotenv"
+	"golang.org/x/time/rate"
+)
+
+const HTMLHeader = "<!DOCTYPE html><html>"
+
+func IsTestRun() bool {
+	return os.Getenv("EXECUTION_ENVIRONMENT") == "test"
+}
+
+func loadEnv() {
+	path := ""
+	if IsTestRun() {
+		path = "../.env"
```

Comment:
Why not move the `.env` file into the test directory so you can delete this special-case code?

Reply:
I think I've found the solution, I moved the env file to the root dir and made the `HandleAuthenticated` function to return a handler and accept the `pwd` and `pass` it should check